e2e-tests/tests/verified-user/case-subscriptions.spec.ts

```
@@ -0,0 +1,225 @@
+    await page.goto("/sign-in");
+
+    const hmctsRadio = page.getByRole("radio", { name: /with a myhmcts account/i });
+    await hmctsRadio.check();
+    await page.getByRole("button", { name: /continue/i }).click();
+
+    await loginWithCftIdam(page, process.env.CFT_VALID_TEST_ACCOUNT!, process.env.CFT_VALID_TEST_ACCOUNT_PASSWORD!);
+
+    await expect(page).toHaveURL(/\/account-home/);
+  });
+
+  test.afterEach(async ({}, testInfo) => {
```

Biome (2.4.13)

[error] Unexpected empty object pattern.

(lint/correctness/noEmptyPattern)